### ISSUE TITLE
Textual Gazetteer Ideas (Ryan Horne)

### ISSUE THREAD

**Issue by pickettj** [pickettj/central_asia_gazetteer#8]
I met with [Ryan Horne]|(https://rmhorne.org/) today and was chatting about our project; he had some helpful ideas for resources to take a look at (i.e. examples of textual and spatial analysis) (attn. @engeljonathan98 ):

- [Recogito](https://recogito.pelagios.org/): You can take a text, run it through their parser, and it recognizes place names.
    - This is only for English texts, so we may not be able to use it but still a cool example.
        - *However,* you can input CSV files, which we could potentially extract from the XML markup.
    - You can even annotate relationships between locations within the text using Recogito.
    - Bottom line: lightweight, online, geographical annotation platform.
- [Article about combining network and geospatial analysis|https://www.academia.edu/1060172/Networks_Territories_and_the_Cartography_of_Ancient_States], I think along the lines of what [Zac](@zme1) was talking about during the meeting.

**Comment by engeljonathan98** [comment]
That networking article was really interesting! Love a good discussion about how power operates, me.

My question is: how do you think this relates to our data? What can we take away from their project for ours, other than potentially networking locations?

To answer that question, I guess I need to know more about what _kind_ of information the gazetteer _has_. What kinds of data does Gulshani provide for each place? Based on the text you've already translated, I can speculate: topographic? Climate? Ethnographic?

I'm gonna make that a separate issue for you to handle.

**Comment by ddaud02** [comment]
This project looks very interesting, I always was fascinated by maps :)
